apps/studio/components/interfaces/Integrations/CronJobs/CronJobScheduleSection.tsx

```diff
@@ -113,13 +113,18 @@ export const CronJobScheduleSection = ({ form }: CronJobScheduleSectionProps) =>
                       value={inputValue}
                       placeholder="E.g. every 5 minutes"
                       className={cn(!useNaturalLanguage && 'hidden')}
+                      onKeyDown={(e) => {
+                        if (e.key === 'Enter') {
+                          e.preventDefault()
+                        }
+                      }}
                       onChange={(e) => setInputValue(e.target.value)}
                     />
                   ) : (
                     <Input_Shadcn_
                       {...field}
                       autoComplete="off"
-                      placeholder="* * * * *"
+                      placeholder="* * * * * *"
```

Comment:
slightly confused here - when should it have 5 and 6 placeholders? from the syntax here it looks like just 5
![image](https://github.com/user-attachments/assets/eff3cb60-7767-456b-ad06-1bc16d4c3c7e)

Reply:
resolving this comment — this is basically the heart of the PR